docs/runbooks/VirtControllerRESTErrorsBurst.md

```diff
@@ -2,8 +2,8 @@
 
 ## Meaning
 
-For the last 10 minutes or longer, over 80% of the REST calls made to
-`virt-controller` pods have failed.
+More than 80% of REST calls in `virt-controller` pods failed in the last 5
+minutes.
```

Comment:
Same comment as on the `VirtApiRESTErrorsBurst.md` file

Reply:
Same

Reply:
https://github.com/kubevirt/monitoring/pull/240#discussion_r1590437383